### ISSUE TITLE
question about generating panoptic-gt

### ISSUE THREAD

**Issue by JoyHuYY1412** [uber-research/UPSNet#27]
https://github.com/uber-research/UPSNet/blob/9191a59de894ae95831b622c7d20102db5c93b6f/upsnet/operators/modules/mask_matching.py#L61

where do you use this function to generate ground truth panoptic logits?  

BTW, is this possible to use COCO panoptic dataset annotations to generate them?

**Comment by YuwenXiong** [comment]
We didn't use this function as we use ground truth roi during training, so https://github.com/uber-research/UPSNet/blob/4da1b65bbd79212ec78f29f359063eb93a8f4ab6/upsnet/operators/modules/mask_matching.py#L28 is enough.

For you second question I'm afraid I don't get it.

**Comment by JoyHuYY1412** [comment]
> We didn't use this function as we use ground truth roi during training, so
> 
> [UPSNet/upsnet/operators/modules/mask_matching.py](https://github.com/uber-research/UPSNet/blob/4da1b65bbd79212ec78f29f359063eb93a8f4ab6/upsnet/operators/modules/mask_matching.py#L28)
> 
> Line 28 in [4da1b65](/uber-research/UPSNet/commit/4da1b65bbd79212ec78f29f359063eb93a8f4ab6)
> 
>  class MaskMatching(nn.Module): 
> is enough.
> For you second question I'm afraid I don't get it.

Thank you for your reply. I mean can we use the annotation images which is already provided in the Coco panoptic dataset to generate the ground truth file you require. 

Btw, does the pretrained model trained on Imagenet for image classification? I load this model and I find the mrcnn accuracy is already very high in the begining, so I wonder has the mrcnn head been trained already.

**Comment by YuwenXiong** [comment]
Currently it does require official coco annotation files.

The pretrained models are pretrained on ImageNet. In joint training setting you may find rcnn accuracy is high at the beginning and will decrease, that is because at the beginning RPN cannot provide proposals with high quality and most proposals are background which are easy to classify.
